apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-notice.tsx

```diff
@@ -14,39 +16,74 @@ const UPGRADE_NOTICE_ID = 'wpcom-live-preview/notice/upgrade';
 
 const unlock = getUnlock();
 
+const LivePreviewUpgradeNoticeView: FC< {
+	noticeText: string;
+	upgradePlan: () => void;
+} > = ( { noticeText, upgradePlan } ) => {
+	return (
+		<Notice
+			status="warning"
+			isDismissible={ false }
+			className="wpcom-live-preview-upgrade-notice-view"
+			actions={ [
+				{
+					// TODO: Add the tracking event.
```

Comment:
The tracks events are probably these:

- calypso_upgrade_nudge_impression
- calypso_upgrade_nudge_cta_click